memoized_lcsh.txt

```diff
@@ -0,0 +1 @@
+{"Semantic Web"=>"sh2002000569"}
```

Comment:
I'm going to update this with what staging returned. I think it makes sense to just use this and remove the spec/fixtures files.